crates/nu-data/src/value.rs

```diff
@@ -182,7 +205,10 @@ pub fn compute_values(
                         }
                         Ok(bigdecimal::BigDecimal::from(x.clone()) % y)
                     }
-
+                    Operator::Pow => {
+                        let yp = bigdecimal::ToPrimitive::to_u32(y).unwrap_or(0);
```

Comment:
same as above: operations on bigint and bigdecimal shouldn't leave the big form and go back to floats and ints.